src/app/models/work-item.ts

```diff
@@ -19,11 +19,11 @@ export class WorkItemAttributes {
 }
 
 export class WorkItemRelations {
-  assignee: {
+  assignees: {
     data: {
       id: string,
       type: string
-    }
+    }[]
```

Comment:
Actually, no. Comments will have a data property which will be an array. I'll add it when I'll be working on comment resolver.